package.json

```diff
@@ -110,6 +110,7 @@
     "passport-local": "^1.0.0",
     "passport-oauth": "^1.0.0",
     "passport-twitter": "^1.0.3",
+    "pm2": "^2.10.2",
```

Comment:
We do not use this flow in development, hence adding this is not needed. However if you are still interested, please move it to dev deps.

Reply:
Ok. I'll move it to dev dependencies.